pkgs/tools/misc/pywb/default.nix

```diff
@@ -0,0 +1,207 @@
+{ lib, fetchFromGitHub, python39, stdenv }:
+
+with lib;
+
+let
+  python = python39.override {
```

Comment:
Are all of these specific version overrides required? Please try to [relax](https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook) the pywb requirements instead, or provide a comment justifying all the package versions.

If not, consider creating all the packages inside the `let ... in`, and remove the python39 override. That allows you to remove the `with python.pkgs;` below

Reply:
I've removed all the overrides, since most of them appear to be there for backwards-compatibility with Python 2. I did have to make a few trivial patches.